_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -44,14 +44,173 @@ The ONT seems only to display output of the ROM CFE and flash CFE, but don't all
 
 {% include serial_dump.html file="fg1000b-11_boot_cfe.txt" alt="Sercomm FG1000B.11 CFE boot dump" title="Sercomm FG1000B.11 CFE boot dump" %}
 
+## Root procedure
+[See how the enable telnet/ssh section](/ont-sercomm-fg1000b-11/#enable-telnetsshserial)
+
 ## List of software versions
 
 Current only version seen is: 090144.1.0.001
 
+## List of partitions
```

Comment:
There are a few improvements we could do for later revisions :
 - adding information about the contents (UBI, JFFS2, squashfs)
 - splitting the virtual mtds : mtd20/21 (gluebi from Rootfs1 or Rootf2) and mtd22 (gluebi from Lib1 or Lib2)